### DISCUSSION TITLE
我八胃大，无需多盐！

### DISCUSSION THREAD

**Discussion by ACGNworld** [share121/inter-knot#504] [其他]
![1721038324739_d](https://github.com/user-attachments/assets/42fb2127-86d2-43e7-a3c4-8fb442ce4e23)
![19033889253782db83432f435693f846587873337](https://github.com/user-attachments/assets/ae709d7f-9e00-4ce1-9308-51834c551414)
老八··嘿嘿嘿···老八

**Comment by Mnpolize** [comment]
味大，无需多盐！！！

**Comment by MIHLFY** [comment]
味大，无需多盐！！！
